ethcore/src/lib.rs

```diff
@@ -57,61 +57,64 @@
 // error_chain foreign_links.
 #![recursion_limit="128"]
 
+extern crate ansi_term;
 extern crate blooms_db;
 extern crate bn;
 extern crate byteorder;
-extern crate crossbeam;
 extern crate common_types as types;
+extern crate crossbeam;
+extern crate ethabi;
 extern crate ethash;
+extern crate ethcore_blockchain as blockchain;
 extern crate ethcore_bloom_journal as bloom_journal;
-extern crate parity_crypto;
+extern crate ethcore_db as db;
 extern crate ethcore_io as io;
-extern crate parity_bytes as bytes;
-extern crate ethcore_logger;
```

Comment:
Running on nightly 1.33 I had to restore this line (extern ethcore_logger) to run test from ethcore.

Reply:
In fact removing #[cfg(feature = 'json-tests'] from the new import works better.

Reply:
1. It's better to run `cargo test -p ethcore` from the maindir (you get the same `Cargo.lock`).
2. It actually requires `#[cfg(any(test, feature = "json-tests"))]`